vulyk/plugins/dummy/__init__.py

```diff
@@ -0,0 +1,30 @@
+import json
+import logging
+
+from werkzeug.utils import import_string
+
+
+logger = logging.getLogger(__name__)
+
+
+def get_task(request):
```

Comment:
What's the difference between this method and `AbstractTaskType::get_next`?
I mean we need to connect module's init and TaskType somehow.
What do you think?
@dchaplinsky ?